### DISCUSSION TITLE
Companies / Sites / Projects using Turborepo

### DISCUSSION THREAD

**Discussion by jaredpalmer** [vercel/turborepo#103] [Show and tell]
If you're using Turborepo at your company/organization you can share the URLs here. If it's open source, share the GitHub link too!

Some sites and projects:

- Vercel - https://vercel.com
- Netflix - https://netflix.com
- Datadog - https://datadoghq.com
- Lattice - https://lattice.com
- TeeSpring - https://spri.ng
- MakeSwift - https://makeswift.com
- On Deck - https://beondeck.com
- Astro - https://astro.build
- Egghead / Skill Recordings - https://github.com/skillrecordings/products
  - Kent C. Dodds Bundle - https://kcdbundle.com
  - Compilers for Humans - https://compilersforhumans.com
  - CSS Destructured - https://cssdestructured.com
  - Keyboard Legend - https://keyboardlegend.com
  - Engineering Management for the Rest of Us - https://engmanagement.dev
  - Escuela Frontend - https://www.escuelafrontend.com/
- Dito - https://www.dito.so/

**Comment by GR34SE** [comment]
🙏 TDSOFT - https://tdsoft.com

**Comment by harisvsulaiman** [comment]
cloodot: https://cloodot.com

**Comment by mfpopa** [comment]
TeamSportz: https://www.teamsportz.pro/ @teamsportz

**Comment by stevejcox** [comment]
https://www.goodrx.com

**Comment by ThibautMarechal** [comment]
🎥 EVS: https://evs.com

**Comment by Aliemeka** [comment]
Onboard: www.onboard.xyz

**Comment by albertilagan** [comment]
https://luxor.tech

**Comment by getanwar** [comment]
Dorik - NoCode Website Building Tool: https://dorik.com

Our Github: https://github.com/dorik

**Comment by nmelo** [comment]
[**Beadbox**](https://beadbox.app) — Native desktop GUI for the [beads](https://github.com/steveyegge/beads) issue tracker. Next.js 16 + React 19 + Tauri v2, bundled with Node.js as a sidecar. We use 8 AI coding agents working simultaneously on the same codebase, coordinated through beads itself.

`brew tap beadbox/cask && brew install --cask beadbox`
